workalendar/tests/test_asia.py

```diff
@@ -82,6 +82,12 @@ def test_year_2013(self):
         self.assertIn(date(2013, 10, 18), holidays)  # eid al adha
         self.assertIn(date(2013, 12, 18), holidays)  # National Day
 
+    def test_weekend(self):
```

Comment:
never mind, you're right.
As soon as the changelog will be amended in your PR and merged, I may start to work on a minor refactor, and adding tests on the ``test_core`` module, but don't worry, your patch is an excellent step in the right direction!